gap/attributes/properties.gi

```diff
@@ -610,12 +615,21 @@ function(S)
   if HasIdempotentGeneratedSubsemigroup(S) or not IsActingSemigroup(S) then
     T := IdempotentGeneratedSubsemigroup(S);
   else
-    min := MinimumList(List(gens, x -> ActionRank(S)(x)));
-    new := Filtered(Idempotents(S), x -> ActionRank(S)(x) >= min);
+    ranks := List(gens, x -> ActionRank(S)(x));
+    min := MinimumList(ranks);
+    if HasIdempotents(S) then
+      new := Filtered(Idempotents(S), x -> ActionRank(S)(x) >= min);
+    else
+      max := MaximumList(ranks);
+      new := [];
+      for i in [min .. max] do
+        Append(new, Idempotents(S, i));
+      od;
+    fi;
     if new = [] then
       return false;
     fi;
-    T := Semigroup(new);
+    T := Semigroup(new, rec(acting := true));
```

Comment:
I see your issue. I did it because of the tests. I wanted to check `IsIdempotentGeneratedSubsemigroup` on a large semigroup which is something like `T10`. The acting code can handle this completely fine, whereas the generic code struggles.

Therefore in the test file I create this example using the `acting := true` option. This is fine when the acting methods are on. However, when we run the tests with `SEMIGROUPS.Test`, in its second pass it sets the options record to have `acting := false`. So although I create the semigroup in the test file as an acting semigroup, and although it happily gets to the line containing `T := Semigroup(new);`, this new semigroup `T` is a generic semigroup which is too big to be computed. Therefore I force it to be acting.